sirepo/package_data/static/js/raydata.js

```diff
@@ -725,13 +821,47 @@ SIREPO.app.directive('scanSelector', function() {
 		);
 	    };
 
+	    $scope.setColumns = function() {
+		$scope.columns = ['select'].concat($scope.defaultColumns.concat(appState.models.metadataColumns.selected));
```

Comment:
I think this is clearer
```js
                $scope.columns = [
                    'selected',
                    ...$scope.defaultColumns,
                    ...appState.models.metadataColumns.selected

                ];
```

Reply:
e64969d8a3a9303ebcf97566971db0411515341f